README.md

```diff
@@ -1,9 +1,6 @@
 Master branch status:
 [![Build Status](https://travis-ci.org/TiBarification/Jail-Warden-Pro.svg?branch=master)](https://travis-ci.org/TiBarification/Jail-Warden-Pro)
 [![CircleCI](https://circleci.com/gh/TiBarification/Jail-Warden-Pro/tree/master.svg?style=svg)](https://circleci.com/gh/TiBarification/Jail-Warden-Pro/tree/master)
-Dev branch status:
-[![Build Status](https://travis-ci.org/TiBarification/Jail-Warden-Pro.svg?branch=dev)](https://travis-ci.org/TiBarification/Jail-Warden-Pro)
-[![CircleCI](https://circleci.com/gh/TiBarification/Jail-Warden-Pro/tree/dev.svg?style=svg)](https://circleci.com/gh/TiBarification/Jail-Warden-Pro/tree/dev)
```

Comment:
I did not have a dev branch and therefore it did not work, but this does not apply to this